src/pactMswAdapter.ts

```diff
@@ -376,6 +376,7 @@ export interface PactInteraction {
     path: string;
     headers: any;
     body: DefaultRequestBody;
+    query: any;
```

Comment:
should be typed as a `string` IMHO

Reply:
It can accept a query object, which is optional. The query object can accept a string, string array. the MatcherResult is irrelevant here as matchers are not supported in this adapters capture mode

```
interface QueryObject {
  [name: string]: string | MatcherResult | string[]
}
export interface RequestOptions {
  method: HTTPMethod | methods
  path: string | MatcherResult
  query?: Query
  headers?: { [name: string]: string | MatcherResult }
  body?: any
}
```

https://github.com/pact-foundation/pact-js/blob/97a06147a69d3896be0b873121f032ef6637f1d1/src/dsl/interaction.ts#L11-L22

json schema for v2 pact

https://bitbucket.org/atlassian/pact-json-schema/src/8d0b9d2f38fe517dfbce0e5d0b8755eaefaacb5b/schemas/v2/schema.json#lines-154

v2 pact specification re query strings

https://github.com/pact-foundation/pact-specification/tree/version-2#request-query-string

Reply:
so I think in our case should just be `query?: string`